src/entities/transaction/model.ts

```diff
@@ -55,3 +56,12 @@ export async function fetchList(params?: PaginationParams): Promise<Paginated<Tr
     data,
   };
 }
+// TODO  paginated List => make generic change a list to paginated list
+export async function fetchBlocksList(selectedTransaction: TransactionDto[], params?: PaginationParams): Promise<Paginated<Transaction>> {
+  const res = pagination(selectedTransaction, params);
+  const data = res.data.map(mapFromDto);
+  return {
+    pagination: res.pagination,
+    data,
+  };
+}
```

Comment:
Apart from other things I have to say about this function, at the very least it has no reason to be `async` - there is no any `await` inside. Making it `async` just brings complexity (because it will return `Promise<T>` instead of just `T`)

```suggestion
export function fetchBlocksList(selectedTransaction: TransactionDto[], params?: PaginationParams): Paginated<Transaction> {
  const res = pagination(selectedTransaction, params);
  const data = res.data.map(mapFromDto);
  return {
    pagination: res.pagination,
    data,
  };
}
```

Reply:
acknowledged and committed 👍🏼